Templating/Helper/BaseHelper.php

```diff
@@ -114,6 +114,25 @@ public static function getICUDataVersion()
             }
         }
 
-        return;
+        return null;
+    }
+
+    /**
+     * https://wiki.php.net/rfc/internal_constructor_behaviour
+     *
+     * @param mixed $instance
```

Comment:
* Missing two params.
* `mixed` ? We don't know which type could be passed at all?